sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,154 @@ def __repr__(self):
         return (f"{self.__class__.__name__}("
                 f"{self.linear_stiffness}, {self.nonlinear_stiffness}, {self.pathway}, "
                 f"equilibrium_length={self.equilibrium_length})")
+
+class CoulombKineticFriction(ForceActuator):
+    """Coulomb kinetic friction with Stribeck and viscous effects.
+
+    Explanation
+    ===========
+
+    This represents a Coulomb kinetic friction with the Stribeck and viscous effect,
+    described by the function:
+
+    .. math::
+        F = f_c sign(v) + (f_{max} - f_c) e^{-(v / v_s)^2} + sigma v
+
+    where :math:`f_c` is the Coulomb friction constant, :math:`f_{max}` is the maximum static friction force,
+    :math:`v_s` is the coefficient of sliding friction, :math:`sigma` is the viscous friction constant,
+    and :math:`v` is the relative velocity.
+
+    The default friction force is :math:`F = mu_k N`, where :math:`N` is the normal force.
+    When specified, the actuator includes:
+
+    - Stribeck effect: :math:`(mu_s - mu_k) N e^{-(v / v_s)^2}`
+    - Viscous effect: :math:`sigma v`
+
+    In case we include all effects, the full actuator represents:
+
+    .. math::
+        F = mu_k N sign(v) + (mu_s - mu_k) N e^{-(v / v_s)^2} + sigma v
+
+    Please note that this actuator assumes slip and applies a force opposite to the velocity direction.
+
+    Parameters
+    ==========
+
+    normal_force : Expr
+        The normal force between the surfaces.
+    mu_k : Expr
+        The coefficient of kinetic friction.
+    pathway : PathwayBase
+        The pathway that the actuator follows.
+    v_s : Expr
+        The coefficient of sliding friction (COF).
```

Comment:
What is the difference in a "coefficient of sliding friction" and a "coefficient of kinetic friction"? I would think they are the same thing.

Reply:
The name for that term varies in different articles, i.e., coefficient of sliding friction, stribeck velocity. I understand it as the maximum static friction coefficient, but in a kinetic friction scenario, it seems reasonable to consider them as the same thing.